packages/core/test/ObjectField.test.jsx

```diff
@@ -394,6 +395,7 @@ describe('ObjectField', () => {
     });
   });
 
+  // TODO label with additionalProperties and arrays
   describe('additionalProperties', () => {
```

Comment:
Whoops, I didn't end up doing that change so removing the TODO
```suggestion
  describe('additionalProperties', () => {
```